tests/test_robotstxt.py

```diff
@@ -224,7 +225,8 @@ def test_access4 (self):
             "Disallow: /tmp",
             "Disallow: /a%3Cd.html",
             "Disallow: /a/b.html",
-            "Disallow: /%7Ejoe/index.html",
+            "Disallow: /%7Ejoe/index.html" if sys.version_info < (3, 7) \
+            else "Disallow: /~joe/index.html",
```

Comment:
You don't need the `\` to continue the line, the surrounding `[`...`]` take care of that.